CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 5.0.5
```

Comment:
At this point, this is more than just a simple bugfix. We should bump this to 5.1.0.